packages/@aws-cdk/aws-eks-v2-alpha/lib/kubectl-handler/index.py

```diff
@@ -0,0 +1,26 @@
+import json
+import logging
+
+from apply import apply_handler
+from helm import helm_handler
+from patch import patch_handler
+from get import get_handler
+
+def handler(event, context):
+  print(json.dumps(dict(event, ResponseURL='...')))
+
+  resource_type = event['ResourceType']
+  if resource_type == 'Custom::AWSCDK-EKS-KubernetesResource':
+    return apply_handler(event, context)
+
+  if resource_type == 'Custom::AWSCDK-EKS-HelmChart':
+    return helm_handler(event, context)
+
+  if resource_type == 'Custom::AWSCDK-EKS-KubernetesPatch':
+    return patch_handler(event, context)
+
+  if resource_type == 'Custom::AWSCDK-EKS-KubernetesObjectValue':
+    return get_handler(event, context)
+
+  raise Exception("unknown resource type %s" % resource_type)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using `Exception` and `BaseException` can make your code prone to errors and difficult to maintain. Instead, we recommend using one of the [Built-in Exceptions](https://docs.python.org/3/library/exceptions.html) or creating a custom exception class that is derived from `Exception` or one of its subclasses.